core/templates/dev/head/components/common-layout-directives/common-elements/attribution-guide.directive.ts

```diff
@@ -33,9 +33,10 @@ angular.module('oppia').directive('attributionGuide', [
         'BrowserCheckerService', 'UrlService', function(
             BrowserCheckerService, UrlService) {
           var ctrl = this;
-          ctrl.isMobileDevice = BrowserCheckerService.isMobileDevice();
-          ctrl.iframed = UrlService.isIframed();
-        }
-      ]
+          ctrl.$onInit = function() {
+            ctrl.isMobileDevice = BrowserCheckerService.isMobileDevice();
```

Comment:
Can we change the attr to ctrl.deviceUsedIsMobile to make clear that it's not a function? (Make sure to change any corresponding usages of the variable, too.)

Reply:
Done.